libmscore/chord.cpp

```diff
@@ -2258,7 +2258,7 @@ void Chord::layoutTablature()
 
       // allocate enough room for glissandi
       if (_endsGlissando) {
-            if (rtick())                        // if not at beginning of measure
+            if (!rtick().isZero())                        // if not at beginning of measure
```

Comment:
Maybe it would be better to overload `operator bool` for `Fraction` class, then it would be possible not to change zero-value checks here and in other places. That probably affects only code style but still may be useful.